seed/seedRole.js

```diff
@@ -0,0 +1,35 @@
+const Role = require('~/models/role')
+const {
+  roles: { STUDENT, MENTOR, ADMIN, SUPERADMIN }
+} = require('~/consts/auth')
+const logger = require('~/logger/logger')
+
+const SeedRole = {
+  createRole: async (role) => {
+    try {
+      const roleToCreate = await Role.create({ value: role })
```

Comment:
just `return Role.create({ value: role })`

Reply:
i did this way to follow the pattern which was used in other methods in the code by somebody else before me, so i do not stand out

Reply:
fixed